### PR TITLE
Feature/listing of users

### PR DESCRIPTION
**Pull request by adam-sas-on**
- Sending package to users
- Use modal on a page with user account
- Filter out employees for specific package/combo

- Apply UX requirements to switch to another page.

### CODE REVIEW

File: onboarding/src/Components/hooks/Packages.js [hunk 1]
```diff
@@ -29,7 +29,7 @@ function Packages(props) {
           isLoaded(true);
           setRows(result);
           const ids = result.map((res) => res.id);
-          const maxId = ids.reduce((a, b) => Math.max(a, b));
+          const maxId = ids.reduce((a, b) => Math.max(a, b));/* Uncaught (in promise) TypeError: reduce of empty array with no initial value; */
```
Comment (annapakula) [inline onboarding/src/Components/hooks/Packages.js:32]: Why not `const maxId = Math.max(...ids);`? ;)

Comment (adam-sas-on) [reply onboarding/src/Components/hooks/Packages.js:32]: I experienced this `typeError` when I didn't have any package or page.

Comment (annapakula) [reply onboarding/src/Components/hooks/Packages.js:32]: Yes, me too, so I propose a little code change like above.

**Review by annapakula — APPROVED** [review]

File: onboarding/src/Components/hooks/Packages.js (hunk 1 above)
Comment (adam-sas-on) [reply onboarding/src/Components/hooks/Packages.js:32]: Any small check if `ids` is not empty should be fine :)

Comment (annapakula) [reply onboarding/src/Components/hooks/Packages.js:32]: Of course, I just think that reduce method is unnecessary here.

**Pull request merged**